### PR TITLE
Modernize ruby code / gem infra, retire BSD support

### PR DESCRIPTION
**Pull request by woodhull**

### CODE REVIEW

File: lib/tokyo_api/client.rb [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
-    alias_method :action_kit, :actionkit
+    alias action_kit actionkit
 
     def bluestatedigital
       @bluestatedigital ||= TokyoApi::Bluestatedigital.new(client: self)
     end
 
-    alias_method :blue_state_digital, :bluestatedigital
+    alias blue_state_digital bluestatedigital
```
Comment (lavaturtle) [inline lib/tokyo_api/client.rb]: should this be removed?

File: lib/tokyo_api/identity.rb [hunk 2]
```diff
@@ -9,50 +11,43 @@ def full_user(id)
...
 
       if with_subscription_status
         params << 'with_subscription_status=true'
-        additional_subscription_parameters = path_for_subscription_status_params(opt_in_public_ids, minimum_consent_level, encrypted)
-        unless additional_subscription_parameters.blank?
-          params << additional_subscription_parameters
-        end
+        additional_subscription_parameters = path_for_subscription_status_params(opt_in_public_ids,
+                                                                                 minimum_consent_level, encrypted)
+        params << additional_subscription_parameters unless additional_subscription_parameters.blank?
```
Comment (lavaturtle) [inline lib/tokyo_api/identity.rb]: Can/should this be
```suggestion
        params << additional_subscription_parameters if additional_subscription_parameters.present?
```

**Review by lavaturtle — APPROVED** [review]
:+1:

File: lib/tokyo_api/identity.rb (hunk 2 above)
Comment (woodhull) [reply lib/tokyo_api/identity.rb]: Yes, under our "prefer positives vs. negatives" style hint

**Pull request merged**